recipes-devtools/valgrind/files/bug344802-unhandled-0xec510f1e.patch

```diff
@@ -1,3 +1,7 @@
+Upstream-Status: Pending
+
+Signed-off-by: Jose Quaresma <jose.quaresma@foundries.io>
```

Comment:
It was sent here https://bugs.kde.org/show_bug.cgi?id=344802, but not really sure that is used by the upstream project. Might be better to say it was proposed and point to this link?

Reply:
done